src/coreComponents/mesh/EmbeddedSurfaceSubRegion.cpp

```diff
@@ -234,9 +206,9 @@ bool EmbeddedSurfaceSubRegion::AddNewEmbeddedSurface ( localIndex const cellInde
       m_toNodesRelation( surfaceIndex, inode ) = elemNodes[inode];
     }
 
-    m_embeddedSurfaceToCell[ surfaceIndex ]      = cellIndex;
-    m_embeddedSurfaceToRegion[ surfaceIndex ]    =  regionIndex;
-    m_embeddedSurfaceToSubRegion[ surfaceIndex ] =  subRegionIndex;
+    m_surfaceElementsToCells.m_toElementIndex[ surfaceIndex ][0]        = cellIndex;
```

Comment:
why the second index?

Reply:
Because I am using the same map both for embeddedElements and faceElements and the face ones are connected to two cells so I think the type of map requires to indexes. It would be nice to remove one for the embedded elems tbh.